packages/examples/with-next/pages/_app.js

```diff
@@ -0,0 +1,49 @@
+import App from "next/app"
+import Head from "next/head"
+import { ConfigProvider, StylesContainer } from "benefit-react"
+
+export default class MyApp extends App {
+  static async getInitialProps({ Component, ctx }) {
+    let pageProps = {}
+
+    if (Component.getInitialProps) {
+      pageProps = await Component.getInitialProps(ctx)
+    }
+
+    return {
+      pageProps: { ...pageProps },
+    }
+  }
```

Comment:
This doesn't seem to be doing anything besides the defaults.  Can this be removed?

Reply:
https://github.com/zeit/next.js#custom-app

Reply:
You’re absolutely right @ericclemmons.